.github/workflows/ci.yml

```diff
@@ -153,7 +153,82 @@ jobs:
           sed -i "s/image: flink:.*/image: ${{ matrix.versions.image }}/" e2e-tests/data/*.yaml
           sed -i "s/flinkVersion: .*/flinkVersion: ${{ matrix.versions.flinkVersion }}/" e2e-tests/data/*.yaml
           git diff HEAD
-          ls e2e-tests/test_*.sh | while read script_test;do \
+          ls e2e-tests/test_application_*.sh e2e-tests/test_sessionjob_*.sh | while read script_test;do \
+            echo "Running $script_test"
+            bash $script_test || exit 1
+          done
+          git reset --hard
+      - name: Stop the operator
+        run: |
+          helm uninstall -n ${{ matrix.config.namespace }} flink-kubernetes-operator
+      - name: Stop minikube
+        run: |
+          source e2e-tests/utils.sh
+          stop_minikube
+  e2e_multi_ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        config:
+          - description: 'WatchNamespaces enabled'
+            namespace: flink
+            extraArgs: '--create-namespace --set "watchNamespaces={default,flink}"'
+        versions:
+          - image: flink:1.13
+            flinkVersion: v1_13
+          - image: flink:1.14
+            flinkVersion: v1_14
+          - image: flink:1.15
+            flinkVersion: v1_15
+    name: e2e_multi_ci
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Cache local Maven repository
+        uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+      - name: Build with Maven
+        run: |
+          set -o pipefail; mvn clean install -DskipTests | tee ./mvn.log; set +o pipefail
```

Comment:
Do we still need `Build with Maven` in the `e2e_multi_ci` ?

Reply:
we do it with every combination, we could probably avoid it, but that needs some extra efforts